lib/icinga/dependency.ti

```diff
@@ -22,6 +22,10 @@ class Dependency : CustomVarObject < DependencyNameComposer
 {
 	load_after Host;
 	load_after Service;
+	load_after Endpoint;
+	load_after CheckCommand;
+	load_after EventCommand;
```

Comment:
What's the reason for these three dependencies? I don't see a reason within this file. If it's for some indirect dependency due to the host or service, these are given as a dependency anyways, so these three are dependencies implicitly already. Unless there's something I'm missing within dependencies themselves, I don't see a reason to make this explicit.